server/src/routes/bots/[id]/index.js

```diff
@@ -167,7 +180,10 @@ module.exports = {
         server_count: 0,
         votes: 0,
         verified: false
-      });
+      };
+      if (webhook?.url && webhook?.token) documentData.webhook = { url: webhook.url, token: webhook.token };
```

Comment:
Instead of checking if there is a webhook url and token and adding it to documentData, you should restore it to its old state and add the following to new Bot:

webhook {
    url: webhook?.url || null,
    token webhook?.token || null
  }